### ISSUE TITLE
Stereo audio channels are swapped

### ISSUE THREAD

**Issue by starquake** [spiritualized1997/openFPGA-Genesis#16]
Stereo audio channels are swapped, left is right and right is left.

Quite weird in Beyond Oasis.

Just try the audio test in 240p Test Suite.

**Comment by IchibanRei** [comment]
Good catch. I just tested this with my Model 1 genesis and left and right channels are swapped on this core.  Also tested in Shining Force, Phantasy Star IV and Alisia Dragoon.  All have the same behavior.
